public/scripts/restaurants.js

```diff
@@ -0,0 +1,53 @@
+var str = (window.location.pathname);
+let restId = str.split('/').slice(-1)[0]
+let notesDisplay = document.getElementById('notesDisplay')
+
+let myMap = L.map('mapTwo').setView([44.4759, -73.2121], 14)
+L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+    maxZoom: 19,
+    attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+}).addTo(myMap)
+
+
+
+// bring in the user selected restaurant info and grab the coords, then drop a pin at those coords..
+fetch(`http://yelpingtonapi.herokuapp.com/api/restaurants/${restId}`)
+    .then(res => res.json())
+    .then((jsonObj) => {
+
+        let coords = jsonObj.coords;
+        var lat = parseFloat(coords[0]);
+        var lon = parseFloat(coords[1]);
+        L.marker([lat, lon]).addTo(myMap)
+    })
+// grab all the values from their keys and show the text in the html where targeted
+fetch(`http://yelpingtonapi.herokuapp.com/api/restaurants/${restId}`)
```

Comment:
Also, worth noting, you make 2 identical `fetch`es to the same endpoint, one after the other.  To save networking power, I encourage you to keep everything within a single request.  That way, after receiving the JSON once, you have all of the information you need to both manipulate the map AND generate the restaurant information.  Super quick refactor, if you're interested!  After all, `restInfo` and `jsonObj` are the same thing.